data/marauders.txt

```diff
@@ -871,7 +871,7 @@ ship "Marauder Raven"
 	explode "tiny explosion" 28
 	explode "small explosion" 40
 	"final explode" "final explosion small"
-	description "The modifier of this Raven has given more heft to an agile and elegant vessel. They've added a turret and some apparently flimsy bulkheads over extra shield emitters and outfit space. If it weren't for the wingtips, you would barely recognize Lionheart's ship under the patchwork that you expect has made an already deadly warship even more so."
+	description "Whoever modified this Raven has given more heft to an agile and elegant vessel. They've added a turret and some apparently-flimsy bulkheads over extra shield emitters and outfit space. If it weren't for the wings, the central hull would barely be recognizeable as Lionheart's original under the patchwork that you expect has made an already deadly warship even more so."
```

Comment:
apparently-flimsy doesn't look right to me.
And "placing bulkheads over shield emitters" sounds a bit strange, as if the shield emitters suddenly have a bulkhead on top of them. (Same for placing bulkheaders over outfit space.)

Reply:
While "apparently flimsy" wasn't wrong, hyphenating it makes it immediately obvious that the word "apparently" is modifying the word "flimsy" and not anything else later in the sentence.

Presumably the bulkheads over the shield emitters are meant to hide them; there's no indication that shield emitters require an unobstructed view of space to operate. I fail to see what's weird about bulkheads covering outfit space. That being said, if you'd rather rewrite the entire description, feel free.